.github/workflows/ci.yaml

```diff
@@ -11,7 +11,7 @@ on:
 env:
```

Comment:
@hj-johannes-lee This commit doesn't seem to depend on another one. I'd propose to submit them in a separate PRs.